dask/local.py

```diff
@@ -404,17 +404,18 @@ def get_async(apply_async, num_workers, dsk, result, cache=None,
         _, _, pretask_cbs, posttask_cbs, _ = unpack_callbacks(callbacks)
         started_cbs = []
         succeeded = False
+        state = {} # if start_state_from_dask fails, we will have something
+                   # to pass to the final block.
```

Comment:
Flake8 seems to dislike the indented comment here. Maybe move the comments above the `state` line?